src/Components/Wizard/Wizard.tsx

```diff
@@ -91,7 +91,7 @@ export class Wizard extends React.Component<WizardProps, WizardState> {
       currentStep: this.currentStep,
       isLastStep: this.isLastStep,
       previous: this.previous,
-      next: this.handleSubmit,
+      next: this.next,
```

Comment:
@erikdstock Do you know why this was mapped to `handleSubmit` initially?

Reply:
Nope - from looking at the blame i think it was changed in https://github.com/artsy/reaction/pull/730- maybe @xtina-starr would have context?

To be honest, as I remember it working on this wizard component was some of my first React+Ts code and I found it all very confusing at the time. As I recall I did a lot of the initial work with guidance from @l2succes but it was Christina + Luc who ended up polishing it off and actually first using the component in login (though I may be wrong).